sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -137,6 +139,22 @@ class JoinSuite extends QueryTest with SharedSQLContext {
     assert(planned.size === 1)
   }
 
+  test("broadcasthint sets relation statistics to smallest value") {
```

Comment:
sorry would be great to put this in the optimizer suites, rather than this file which is an end-to-end suite.

I just took a look at the suites available. I'd rename JoinOrderSuite to JoinOptimizationSuite, and then put this case there.